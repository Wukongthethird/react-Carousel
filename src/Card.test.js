import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card  from "./Card";

it("Displays Card on DOM", function() {
  const { container, debug } = render(<Card caption="Rithm" src="School" currNum={1} totalNum={4}/>);
  debug(container);

  expect(container).toMatchSnapshot();
  expect(container.querySelector(".Card-small")).toContainHTML("Image" );
  expect(container.querySelector(".Card")).toContainHTML("School" );
  expect(container.querySelector(".Card-small")).not.toContainHTML("School");
});

