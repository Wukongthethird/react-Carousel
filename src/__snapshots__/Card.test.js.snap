// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays Card on DOM 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Rithm
    </h4>
    <img
      alt="Rithm"
      class="Card-image"
      src="School"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      4
      .
    </small>
  </div>
</div>
`;
